Introduce option to enable wiki links processing (#39)
wikimedia/banana-i18n: README.md

## Grammar

```javascript

> const banana=new Banana( 'fi' );

Reviewer: ```suggestion
const banana = new Banana('fi');
```

The embedded context's directionality is determined by looking at the argument for `$1`, and then explicitly inserted into the Unicode text, ensuring correct rendering (because then the bidi algorithm "knows" the argument text is a separate context).

## Wiki style links

> The message can have [mediawiki markup style links](https://en.wikipedia.org/wiki/Help:Wikitext#Links_and_URLs). By default this is disabled. To enables support for this, pass `wikilinks=true` option to `Banana` constructor. Example:

Reviewer: ```suggestion
The message can use [MediaWiki link syntax](https://www.mediawiki.org/wiki/Help:Links). By default this is disabled. To enable support for this, pass `wikilinks=true` option to `Banana` constructor. Example:
```

```

> new Banana('es', { wikilinks: true } )

Reviewer: This line still has inconsistent whitespace.

> new Banana('es', { wikilinks:true } )

Reviewer: Inconsistent spacing.

```suggestion
new Banana('es', { wikilinks: true })
```

```

The original wiki links markup is elaborate, but here we only support simple syntax.

> * Internal links:  `[[pageTitle]]`  or `[[pageTitle|displayText]]`

Reviewer: Does this give wiki/ prefix or how it works in this library? Maybe add an example output too to clarify.

Author: It is relative links `<a href="./Apple" title="Apple">Apple</a>`

> * External links: `[externalURL]` or `[externalURL displayText]`

Reviewer: ```suggestion
* External links: `[https://example.com]` or `[https://example.com display text]`
```
